core/ai-commit-msg/skills/ai-commit-msg/SKILL.md

```
@@ -0,0 +1,206 @@
+
+Writes an AI-generated conventional commit message to the `COMMIT_EDITMSG` file via the `prepare-commit-msg` Git hook. Reads the staged diff and passes it to your configured AI provider.
+
+## Setup
+
+1. Install and initialise husky:
+
+```sh
+npm install --save-dev husky && npx husky init
+```
+
+2. Create `.husky/prepare-commit-msg`:
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)